sklearn/preprocessing/tests/test_function_transformer.py

```diff
@@ -74,6 +74,12 @@ def test_delegate_to_func():
     )
 
 
+def test_get_feature_names():
+    F = FunctionTransformer()
+    feature_names = F.get_feature_names(["a", "b", "c"])
+    testing.assert_array_equal(['a', 'b', 'c'], feature_names)
+
```

Comment:
Need to test handling where `input_features` is not given explicitly. Indeed, we may need to decide whether to return None or to construct a default set of feature names is the better option.